content/en/changelogs/1.32.0-changelog.md

```diff
@@ -13,7 +13,7 @@ This release includes fixes, features, and performance improvements across a wid
 
 ### Java 17
 
-Front50 and Igor containers now default to running with the Java 17 JRE. In an upcoming release (likely 1.33) all other services will default to the Java 17 JRE. If you encounter issues with these containers you can opt-out and use the Java 11 JRE containers. Users of Halyard can do this by specying `imageVariant: java11-slim` or `imageVariant: java11-slim-ubuntu` in their halconfig.
+Front50 and Igor containers now default to running with the Java 17 JRE. In an upcoming release (likely 1.33) all other services will default to the Java 17 JRE. If you encounter issues with these containers you can opt-out and use the Java 11 JRE containers. Users of Halyard can do this by specying `imageVariant: JAVA11` or `imageVariant: UBUNTU_JAVA11` in their halconfig.
```

Comment:
I approved this, but can you also add a note that the minimum halyard version to support this is 1.62.0?